plugins/basemap-gaode-by-gmoogway.user.js

```diff
@@ -0,0 +1,351 @@
+// ==UserScript==
+// @id             iitc-plugin-basemap-gaode-by-gmoogway
+// @name           IITC plugin: Map layers from GAODE by GMOogway
+// @category       Map Tiles
+// @version        0.2.0.@@DATETIMEVERSION@@
+// @description    [@@BUILDNAME@@-@@BUILDDATE@@] Add autonavi.com (China) map layers
+@@METAINFO@@
+// ==/UserScript==
+
+@@PLUGINSTART@@
+
+    // PLUGIN START ////////////////////////////////////////////////////////
```

Comment:
Here should not be any indent, see other iitc plugins

Reply:
fixed